forte/integrals/integrals.h

```diff
@@ -262,6 +263,10 @@ class ForteIntegrals {
     /// @param Db The beta 1RDM (nactv x nactv, no symmetry) from RDMs class
     virtual void make_fock_matrix(ambit::Tensor Da, ambit::Tensor Db) = 0;
 
+    /// Make the generalized Fock matrix directly from integral values, avoid using JK builder
+    virtual void make_fock_matrix_from_value(std::shared_ptr<psi::Matrix> gamma_a,
```

Comment:
This may not be a pure virtual. We could throw a "not overloaded" error inside the function.